.github/workflows/test.yml

```diff
@@ -0,0 +1,12 @@
+name: 'Test'
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: self-hosted
```

Reply:
Why actually?

Reply:
You can leave it, these small ones are not a real issue